app/index.js

```diff
@@ -294,3 +294,28 @@ app.on('will-quit', () => {
     global.IPFS_PROCESS.kill()
   }
 })
+
+app.on('window-all-closed', () => {
+  // On MacOS this is already the expected behavior, no need to alert the user/close the app
+  if (process.platform === 'darwin') return
+
+  const systemTrayNotification = Settings.getSync('systemTrayNotification')
+
+  if (!systemTrayNotification) {
```

Comment:
This will trigger the message every time the settings is `false`, try with a more specific setup:
`systemTrayNotification === undefined`